src/widget/weffectparameternamebase.cpp

```diff
@@ -66,6 +82,10 @@ void WEffectParameterNameBase::parameterUpdated() {
         m_text = kNoEffectString;
         setBaseTooltip(tr("No effect loaded."));
     }
+    optimumWidth = math_max(
+            optimumWidth,
+            metrics.size(0, m_text).width());
+    m_widthHint = optimumWidth;
```

Comment:
let's rename `optimumWidth` to `valueWidth` and 
```suggestion
    m_widthHint = math_max(
            valueWidth,
            metrics.size(0, m_text).width());
```
and remove the debug ouput below